website/docs/jellyfish/api/spv.md

```diff
@@ -216,3 +216,89 @@ interface CreateAnchorResult {
   sendMessage: string
 }
 ```
+
+## listAnchors
+
+List anchors.
+
+```ts title=client.spv.listAnchors()"
+interface spv {
+  listAnchors (
+    options: ListAnchorsOptions = { minBtcHeight: -1, maxBtcHeight: -1, minConfs: -1, maxConfs: -1 }
```

Comment:
should this one be like in the implementation? but this one gives extra details about the default values🤔

Reply:
Let's keep this, it gives extra details